frontend/packages/dev-console/src/components/pipelineruns/logs/Logs.scss

```diff
@@ -0,0 +1,12 @@
+.odc-logs {
```

Comment:
Should the file and classes be named to something more specific than only `Logs` as we have other Log components